src/Digdir.Domain.Dialogporten.WebApi/Endpoints/V1/ServiceOwner/Dialogs/Update/UpdateDialogSwaggerConfig.cs

```diff
@@ -1,5 +1,7 @@
 using System.Globalization;
+using Digdir.Domain.Dialogporten.Application.Features.V1.Common.Actors;
```

Comment:
Hvorfor er denne her fremdeles? I mitt hode burde ikke dette namespacet eksistere lenger?

Reply:
Den usingen var ikke nødvendig, men ActorValidationErrorMessages er fortsatt i det namespacet

Reply:
Ser at ActorValidationErrorMessages blir bare brukt i den ny actorValidatoren for SO, men errorMessage ligger i v1 features common. Om/Når det blir Enduser Actor validering så vil den nok brukes av begge.